### PR TITLE
'Meal types' -> 'Category', and adding Backing as new category

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala [hunk 1]
```diff
@@ -102,7 +102,7 @@ case class Tag(
 
 object Tag {
   val cuisines = Seq("African", "British", "Caribbean", "French", "Greek", "Indian", "Irish", "Italian", "Japanese", "Mexican", "Nordic", "North African", "Portuguese", "South American", "Spanish", "Thai and South East Asian")
-  val mealTypes = Seq("Barbecue", "Breakfast", "Budget", "Canapes", "Dessert", "Dinner party", "Drinks and cockails", "Healthy eating", "Lunch", "Main course", "Picnic", "Sides", "Snacks", "Starters")
+  val mealTypes = Seq("Baking", "Barbecue", "Breakfast", "Budget", "Canapes", "Dessert", "Dinner party", "Drinks and cockails", "Healthy eating", "Lunch", "Main course", "Picnic", "Sides", "Snacks", "Starters")
```
Comment (tsop14) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: I'd change this `val mealTypes` to `val Category` for consistency.

File: ui/app/com/gu/recipeasy/views/recipe.scala.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
                     curatedRecipeForm("steps"), curatedRecipeForm
                     )('_label -> "Method steps")
                 @bootstrap3.multiTag(curatedRecipeForm("tags")("cuisine"), curatedRecipeForm)(Tag.cuisines, "cuisine", '_label -> "Cuisines")
-                @bootstrap3.multiTag(curatedRecipeForm("tags")("mealType"), curatedRecipeForm)(Tag.mealTypes, "mealType", '_label -> "Meal types")
+                @bootstrap3.multiTag(curatedRecipeForm("tags")("mealType"), curatedRecipeForm)(Tag.mealTypes, "mealType", '_label -> "Category")
```
Comment (tsop14) [inline ui/app/com/gu/recipeasy/views/recipe.scala.html]: `("tags")("mealType")` `("tags")("category")`

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala (hunk 1 above)
Comment (shtukas) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: Cool. I was wondering what you though about it.

**Comment by tsop14** [comment]
https://github.com/guardian/recipeasy/blob/ph-20160929-update-in-mealtypes/ui/app/com/gu/recipeasy/controllers/Application.scala#L88 does this need to change too?

**Comment by shtukas** [comment]
Good catch!

**Comment by shtukas** [comment]
Done!

**Comment by tsop14** [comment]
👍

**Pull request merged**